domains/transfer/Transfer.API/API.IntegrationTests/Migrations/ExampledMigrationTests.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using API.Data;
+using API.IntegrationTests.Factories;
+using API.Models;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.Extensions.DependencyInjection;
+using Npgsql;
+using Xunit;
+
+namespace API.IntegrationTests.Migrations;
+
+public class ExampledMigrationTests : IClassFixture<TransferAgreementsApiWebApplicationFactory>
+{
+    private readonly TransferAgreementsApiWebApplicationFactory factory;
+
+    public ExampledMigrationTests(TransferAgreementsApiWebApplicationFactory factory)
```

Comment:
Introduce a class for a PostgresContainer and use that instead of the factory

Reply:
corrected